modules/update-using-custom-machine-config-pools-mcp-remove.adoc

```diff
@@ -5,14 +5,14 @@
 [id="update-using-custom-machine-config-pools-mcp-remove_{context}"]
 = Moving a node to the original machine config pool
 
-In this canary rollout update process, after you have unpaused a custom machine config pool (MCP) and verified that the applications on the nodes associated with that MCP are working as expected, you should move the node back to its original MCP by removing the custom label you added to the node.
+After you update and verify applications on nodes in a custom machine config pool (MCP), move the nodes back to their original MCP by removing the custom label you added to the nodes.
 
 [IMPORTANT]
 ====
 A node must have a role to be properly functioning in the cluster.
 ====
 
```

Comment:
I honestly go back and forth sometimes about whether there should be prerequisites for a procedure that include basically "accomplishment of the task in the previous section". Idk how well we adhere to our mod docs guidelines in reviews, but the [prereqs section](https://redhat-documentation.github.io/modular-docs/#prerequisites) states to "not list obvious prerequisites".

In this case, not only is the general workflow of described further up in the assembly, I would argue that any user following along in this assembly would see that there's no point in doing this last procedure unless they've accomplished the rest of the stuff preceding this. Let me know what you think, I can add a prereqs section if you think it would be helpful.

Reply:
No including it is fine with me.